### ISSUE TITLE
m3u files question

### ISSUE THREAD

**Issue by spotanjo3** [4gray/iptvnator#115]
When I add via file upload then it's running just fine but I just want to save m3u file to my PC hard drive. How ? VLC is able to do that.

**Comment by what-a** [comment]
One can add a playlist "via file upload" from a location on your PC, or "via URL" from a location on the internet. So, if you are adding a playlist "via file upload" then it is already on your PC, right?

**Comment by 4gray** [comment]
Yeah, just use your browser to download the file. At the moment there is no export function in the application. This is because playlists are only converted one way from XML to JSON and not back.
